docs/virtual_machines/instancetypes.md

```diff
@@ -238,15 +239,164 @@ virtualmachine.kubevirt.io "vm-cirros-csmall" deleted
 
 $ kubectl get controllerrevision/controllerrevision/vm-cirros-csmall-csmall-72c3a35b-6e18-487d-bebf-f73c7d4f4a40-1
 Error from server (NotFound): controllerrevisions.apps "vm-cirros-csmall-csmall-72c3a35b-6e18-487d-bebf-f73c7d4f4a40-1" not found
+```
+
+Users can opt-in to moving to a newer generation of an instance type or preference by removing the referenced `revisionName` from the appropriate `matcher` within the `VirtualMachine` object. This will result in fresh `ControllerRevisions` being captured and used, For example:
+
+```yaml
+$ kubectl apply -f examples/csmall.yaml -f examples/vm-cirros-csmall.yaml
+virtualmachineinstancetype.instancetype.kubevirt.io/csmall created
+virtualmachine.kubevirt.io/vm-cirros-csmall created
+
+$ kubectl get vm/vm-cirros-csmall -o json | jq .spec.instancetype
+{
+  "kind": "VirtualMachineInstancetype",
+  "name": "csmall",
+  "revisionName": "vm-cirros-csmall-csmall-3e86e367-9cd7-4426-9507-b14c27a08671-1"
+}
+
+$ virtctl start vm-cirros-csmall
+VM vm-cirros-csmall was scheduled to start
+
+$ kubectl get vmi/vm-cirros-csmall -o json | jq .spec.domain.cpu
+{
+  "cores": 1,
+  "model": "host-model",
+  "sockets": 1,
+  "threads": 1
+}
+
+$ virtctl stop vm-cirros-csmall
+VM vm-cirros-csmall was scheduled to stop
+
+$ kubectl patch VirtualMachineInstancetype/csmall --type merge -p '{"spec":{"cpu":{"guest":2}}}'
+virtualmachineinstancetype.instancetype.kubevirt.io/csmall patched
 
+$ kubectl patch vm/vm-cirros-csmall --type merge -p '{"spec":{"instancetype":{"revisionName":""}}}'
+virtualmachine.kubevirt.io/vm-cirros-csmall patched
+
+$ kubectl get vm/vm-cirros-csmall -o json | jq .spec.instancetype
+{
+  "kind": "VirtualMachineInstancetype",
+  "name": "csmall",
+  "revisionName": "vm-cirros-csmall-csmall-3e86e367-9cd7-4426-9507-b14c27a08671-2"
```

Comment:
Might add a short note that the `1` in the `revisionName` changed to `2`, otherwise hard to read.